feature-group-definitions/streams.dist.yml

```diff
@@ -0,0 +1,89 @@
+# Generated from: streams.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: Streams
+description: The streams API creates, composes, and consumes continuously generated data.
+spec: https://streams.spec.whatwg.org/
+caniuse: streams
+status:
+  baseline: low
+  baseline_low_date: 2023-03-27
+  support:
+    chrome: "98"
```

Comment:
OK, got it. We can't exactly match that with BCD (I suspect because caniuse doesn't note the missing `WritableStream` interface in Safari before 14.5), but we can get pretty close. See the latest diff.

Reply:
Thanks @ddbeck! Merged!